lib/plaid/products/asset_report.rb

```diff
@@ -94,6 +143,14 @@ class AssetReportCreateResponse < Models::BaseResponse
     property :asset_report_id
   end
 
+  # Public: The response wrapper for /asset_report/filter.
+  class AssetReportFilterResponse < AssetReportCreateResponse
```

Comment:
Do we have any known fields here?

Reply:
I'm not sure what you mean. `AssetReportFilterResponse` is the same as `AssetReportCreateResponse`. Is there a better way to represent this?

Reply:
I see. Just missed the fact that `AssetReportCreateResponse` already has fields defined.